pages/governance.md

```diff
@@ -0,0 +1,291 @@
+reflect the values of our community. Both these links have information on how to
+confidentially report a violation.
+
+## Entities
+
+This section outlines the different entities that exist within the JuMP project,
+their basic role, and how membership of each entity is determined.
+
+### Benevolent Dictator For Life
+
+Due to his role in the creation of JuMP, [Miles Lubin](https://github.com/mlubin)
+holds the role of [Benevolent Dictator For Life (BDFL)](https://en.wikipedia.org/wiki/Benevolent_dictator_for_life).
+
+### Core Contributors
+
+Core contributors lead the technical development of the JuMP project, and they
+are the ultimate authority on the direction of the JuMP project.
+
+The current core contributors are:
+
+* Miles Lubin (@mlubin)
+* Benoît Legat (@blegat)
+* Joaquim Dias Garcia (@joaquimg)
+* Joey Huchette (@joehuchette)
+* Oscar Dowson (@odow)
+
+A new core contributor may be added by consensus of the current core
+contributors and notification to the [Steering Council].
+
+Before becoming a core contributor, it is expected that the community member is
+a [repository maintainer](#repository-maintainers) of several repositories in
+the [JuMP-dev Github organization].
+
+### Emeritus Core Contributors
+
+Emeritus core contributors are community members who were [core contributors],
+but have stepped back to a less active role.
+
+The emeritus core contributors are:
+
+* Iain Dunning (@iainnz)
+
+A [core contributor](#core-contributors) may choose to switch to emeritus status
+by informing the other [core contributors] and the [Steering Council].
+
+### Repository Maintainers
+
+Repository maintainers are trusted members of the community who help the [core
+contributors] by managing a limited number of repositories in the [JuMP-dev
+Github organization].
+
+Any [core contributor](#core-contributors) has individual authority to appoint a
+community member as a repository maintainer of a repository with notification to
+the other [core contributors].
+
+Before becoming a repository maintainer, it is expected that the community
+member will have been an active participant in the development and maintenance
+of a repository for a sustained period of time. This includes triaging issues,
+proposing and reviewing pull requests, and updating any binary dependencies as
+needed.
+
+Community members may be maintainers of multiple repositories at the same time.
+
+Repository maintainers can step back from their role at any time by informing a
+[core contributor](#core-contributors). Furthermore, [core contributors], by
+consensus, can choose to remove repository maintainers for any reason (typically
+inactivity).
+
+### NumFOCUS
+
+JuMP is a [fiscally sponsored project of NumFOCUS](https://numfocus.org/project/jump),
+a 501(c)(3) public charity in the United States. NumFOCUS provides JuMP with
+fiscal, legal, and administrative support to help ensure the health and
+sustainability of the project.
+
+### Steering Council
```

Comment:
The name of the list is jump-steering-committee@...
Did we intend to rename the committee to a council?

Reply:
> Did we intend to rename the committee to a council?

I copied the SciML language, and couldn't remember if it was a council or a committee. Will change.